index.d.ts

```diff
@@ -239,12 +239,12 @@ export interface FastifyGQLCommonOptions {
   };
 }
 
-export type FastifyGQLOptions = FastifyGQLCommonOptions & (FastifyGQLGatewayOptions | FastifyGQLSchemaOptions)
+export type MercuriusOptions = MercuriusCommonOptions & (MercuriusGatewayOptions | MercuriusSchemaOptions)
 
 declare function fastifyGQL
```

Comment:
`fastifyGQL` should be renamed to `mercurius` for consistency

Reply:
Sorry I missed this, I'm doing this change on master.